hudi-common/src/main/java/org/apache/hudi/common/table/log/AbstractHoodieLogRecordReader.java

```diff
@@ -817,6 +805,8 @@ public abstract static class Builder {
 
     public abstract Builder withReaderSchema(Schema schema);
 
+    public abstract Builder withReaderSchema(InternalSchema internalSchema);
+
```

Comment:
We should not keep 2 `withReaderSchema` here, keeps either one of them.

Reply:
removed `withReaderSchema(Schema schema)`